README.md

```diff
@@ -127,7 +127,9 @@ cd bosh-lite
  - Use VM_MEMORY=15000 if you want to host 3 VMRs that can form an HA Group
 
 ~~~~
-VM_MEMORY=5000 vagrant up --provider=virtualbox
+Linux: VM_MEMORY=5000 vagrant up --provider=virtualbox
+Windows: set VM_MEMORY=5000
+         vagrant up --provider=virtualbox
 ~~~~
```

Comment:
How about having two code blocks. One for Linux and one for Windows instead of imbedding the OS into the code block?